e2e/test/scenarios/permissions/sandboxing/helpers/e2e-sandboxing-helpers.ts

```diff
@@ -20,23 +20,36 @@ const { H } = cy;
 const { ALL_USERS_GROUP, DATA_GROUP, COLLECTION_GROUP } = USER_GROUPS;
 const { PRODUCTS_ID, ORDERS_ID, ORDERS, PRODUCTS } = SAMPLE_DATABASE;
 
-type CustomColumnType = "boolean" | "string" | "number";
+type CustomColumnType =
+  | "numberExpr"
+  | "numberLiteral"
+  | "stringExpr"
+  | "stringLiteral"
+  | "booleanExpr"
+  | "booleanLiteral";
```

Comment:
This should probably be

```suggestion
type CustomColumnType = keyof typeof customColumnTypeToFormula
```

And 

```
const customColumnTypeToFormula = {
  boolean: '[Category]="Gizmo"',
  booleanExpr: '[Category]="Gizmo"',
  string: 'concat("Category is ",[Category])',
  stringExpr: 'concat("Category is ",[Category])',
  number: 'if([Category] = "Gizmo", 1, 0)',
  numberExpr: 'if([Category] = "Gizmo", 1, 0)',
  booleanLiteral: "true",
  stringLiteral: '"fixed literal string"',
  numberLiteral: "1",
} as const;
```

To avoid having to duplicate the keys.

Reply:
Thanks. Fixed in latest commit: https://github.com/metabase/metabase/pull/56190/commits/70d769e6250cdd8b837889b3b78f25afefa60420